torch/distributed/nn/api/remote_module.py

```diff
@@ -69,16 +69,20 @@ def _instantiate_template(module_interface_cls, enable_moving_cpu_tensors_to_cud
     )
 
 
-def _create_module(module_cls, args, kwargs, device="cpu", module_interface_cls=None):
+def _create_module(module_cls, args, kwargs, device):
     module = module_cls(*args, **kwargs)
     if not isinstance(module, nn.Module):
         raise ValueError(
             "Expect `module_cls(*args, **kwargs)` returns an instance of <class nn.Module>, "
             f"but it returns an instance of {type(module)}."
         )
+    module.to(device)
+    return module
+
+def _create_module_with_interface(module_cls, args, kwargs, device, module_interface_cls):
```

Comment:
Still keep the default values?

Reply:
this is only called internally and value is provided, any reason to keep it? note this is not even the same method.